src/ccutil/unicharset.h

```diff
@@ -573,13 +573,13 @@ class UNICHARSET {
                       int min_bottom, int max_bottom,
                       int min_top, int max_top) {
     unichars[unichar_id].properties.min_bottom =
-        ClipToRange<int>(min_bottom, 0, UINT8_MAX);
+        static_cast<uint8_t>(ClipToRange<int>(min_bottom, 0, UINT8_MAX));
```

Comment:
This is a typical example where the type casts hide a problem in the code. The values which are clipped here are read from a file. Simply clipping them to 0...255 hides problems in that file which should only provide values in that range.

Therefore I consider such type casts as unwanted.

General rule: Don't add a type cast just to silence a compiler warning.

Reply:
I now understand the rules for casting.  Thank you so much.